database/src/main/java/com/decathlon/ara/repository/custom/impl/ErrorRepositoryImpl.java

```diff
@@ -56,7 +56,7 @@
 @Slf4j
 public class ErrorRepositoryImpl implements ErrorRepositoryCustom {
 
-    private static final Sort ERROR_SORTING = new Sort(Sort.Direction.ASC,
+    private static final Sort ERROR_SORTING = new Sort(Sort.Direction.DESC,
```

Comment:
I've ran your branch locally and still have the errors displayed in Ascending order.
<img width="209" alt="Screenshot 2019-10-21 at 16 13 55" src="https://user-images.githubusercontent.com/45944760/67213198-e2559800-f41d-11e9-9d19-5f6bf8a56c00.png">
<img width="1009" alt="Screenshot 2019-10-21 at 16 14 27" src="https://user-images.githubusercontent.com/45944760/67213199-e2ee2e80-f41d-11e9-9279-0f26a8388e71.png">


Could you please check if you reproduce the old behavior on your side ?